README.md

```diff
@@ -50,28 +61,18 @@ Source files for all Knative community and governance topics are located
 separately in the [knative/community](https://github.com/knative/community/)
 repo.
 
-### Documentation versions for Knative releases
-
-Each release of the Knative docs are available in the website (starting with
-0.3) and their source files are all stored in
-[branches of this repo](doc-releases.md).
-
-## Contributing
-
-We're excited that you're interested in contributing to the Knative documentation! Check out the resources below to get started.
-
-## Getting started
+To help you get started, see the following resources:
 
-- [Knative contributor's guide](help/contributing) -- Contains information about how
+- [Knative docs contributor's guide](https://knative.dev/docs/help/contributor/) -- Contains information about how
```

Comment:
```suggestion
- [Knative docs contributor's guide](https://knative.dev/docs/help/contributor/README.md) -- Contains information about how
```

Reply:
Adding `README.md` to this link does not work. We only need to add `.md` to relative links not absolute links. 

Generally we should be using relative links, but I have used an absolute link here because this README just for GitHub and is not on the website. I don't want it to link to the page in GitHub -- I want it to link to the website page.

Reply:
Ah, okay.